src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -70,3 +70,7 @@ export const ATTRIBUTE_NAME_SOURCE = 'source';
 export const KEY_CODE_FOR_ENTER = 13;
 export const KEY_CODE_FOR_ESCAPE = 27;
 export const KEY_CODE_FOR_SPACE = 32;
+export const mapSources = {
+  'MLG': 'Mulago Hospital (Dictionary)',
```

Comment:
Is this only for testing purposes?

Reply:
Hi Daniel, I have used it in `src/components/dictionaryConcepts/components/AnswerRow.jsx` Instead of having to hard code the sources and a conditional statement in that file, I created a dictionary of the possible sources in this file, imported it to the `AnswerRow` and then simply fetch corresponding value of the source name.

Reply:
Is Mulago Hospital also a source to be used by Andy? I thought by doing this you are already hard coding.

Reply:
Okay I now get it, let me change that

Reply:
I've changed it to display the `currentDictionary` - which is the dictionary containing the concept being edited, as it was before. Is it okay?